tests/mocks/dynamodb.py

```diff
@@ -167,25 +167,92 @@ def dynamodb_create_table_with_range_key(dynamodb_client):
 
 
 @pytest.fixture
-def dynamodb_put_item(dynamodb_create_table, dynamodb_client):
-    table_connection = dynamodb_client.Table("movies-prd")
+def dynamodb_put_item(dynamodb_create_table, dynamodb_resource):
+    table_connection = dynamodb_resource.Table("movies-prd")
     table_connection.put_item(
         Item={"movieName": "The Dark Knight", "year": "2008", "genre": "action"}
     )
     yield
 
 
 @pytest.fixture
-def dynamodb_put_item_for_query(dynamodb_create_table_query, dynamodb_client):
-    table_connection = dynamodb_client.Table("players-prd")
+def dynamodb_put_item_for_query(dynamodb_create_table_query, dynamodb_resource):
+    table_connection = dynamodb_resource.Table("players-prd")
     table_connection.put_item(Item={"name": "inQdo", "number": "5"})
     yield
 
 
 @pytest.fixture
-def dynamodb_put_item_with_range(dynamodb_create_table_with_range_key, dynamodb_client):
-    table_connection = dynamodb_client.Table("movies-prd")
+def dynamodb_put_item_with_range(dynamodb_create_table_with_range_key, dynamodb_resource):
+    table_connection = dynamodb_resource.Table("movies-prd")
     table_connection.put_item(
         Item={"movieName": "The Dark Knight", "year": "2008", "genre": "action"}
     )
     yield
+
+
+@pytest.fixture()
+def dynamodb_client(aws_credentials):
```

Comment:
It may be more convenient to move this code to the top of the file, next to the dynamodb_resource declaration.